From b13ec589a229c9ac044d1b31de18da1f9129008b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 25 Nov 2018 18:53:14 +0200
Subject: [PATCH] preboot: use absolute path for global includes

The hl_regs.h is located on the global scope and therefore should be
refered by absolute include path (e.g. '<>') instead of relative.

Change-Id: Ic73f2009d1068172473d0e22846a5676b1cfa699
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot.c b/preboot.c
index bdc8e88..40eb772 100644
--- a/preboot.c
+++ b/preboot.c
@@ -9,7 +9,7 @@
 #include <common.h>
 #include <linux/io.h>
 #include <linux/sizes.h>
-#include "mach/hl_regs.h"
+#include <mach/hl_regs.h>
 #include "auto_generated.h"
 #include <stddef.h>
 #include <stdbool.h>
